docs/graph_explorer.html

```diff
@@ -382,7 +384,13 @@
 
         `.trim();
 
-
+        const loadMachineConfig = defaultConfig =>{
+          const urlArg                = new URLSearchParams(window.location.search);
+          const compressed            = urlArg.get('s');
+          const sharedMachineConfig   = compressed ? LZString.decompressFromEncodedURIComponent(compressed) : null;
+          const machineConfig         = sharedMachineConfig || defaultConfig;
+          lconfig                     = machineConfig;
+        }
```

Comment:
so nothing calls `loadMachineConfig` at this point

Reply:
probably the thing to do here is to return `machineConfig` as a value from the function